### DISCUSSION TITLE
blog/about/

### DISCUSSION THREAD

**Discussion by giscus** [Vanilla-Yukirin/BlogComments#11] [Announcements]
# blog/about/

我叫 $Vanilla$ ，或者Vanilla_chan，是一个AFOIer & ACMer & MCMer。 现大一，就读于CJLU大数据专业。 是一个蒟蒻，以前是，现在还是，可能永远都是。 但是我会努力的。 另外，希望再可爱一点awa 喜欢的事情 打GalGame 折腾东西，但其实啥都不精通

https://vanilla-chan.cn/blog/about/

**Comment by ghost** [comment]
hsh太有实力啦

**Comment by man0521** [comment]
ただいま~

**Reply by Vanilla-Yukirin** [reply]
お帰りなさい~

**Comment by PerfectLifeG** [comment]
大一这么屌，nb

**Reply by Vanilla-Yukirin** [reply]
谢谢您的夸奖！

**Comment by ArisuMika520** [comment]
## 还在**大二**呢，~~老东西快下来~~，轮到我当大二了

**Reply by Vanilla-Yukirin** [reply]
已严肃升级为大三老登

**Comment by Yohane-Mashiro** [comment]
# 我去，是ACM爷

**Reply by Vanilla-Yukirin** [reply]
<img src="https://vanilla-picture-with-picgo.oss-cn-shenzhen.aliyuncs.com/img-from-OSS-uploader/20260308-211403-10001.png" alt="描述" width="200" height="200" />
